static/app/utils/docs.tsx

```diff
@@ -58,3 +58,10 @@ export function getDocsPlatform(
   // can't find a matching docs platform
   return null;
 }
+
+export function getConfigureTracingDocsLink(platform: string | null): string | null {
+  const docsPlatform = platform ? getDocsPlatform(platform, true) : null;
+  return docsPlatform === null
+    ? null // this platform does not support performance
+    : `https://docs.sentry.io/platforms/${docsPlatform}/performance/`;
+}
```

Comment:
```suggestion
export function docsUrl(strings, platform, ...args) {
  const docsPlatform = platform ? getDocsPlatform(platform, true) : null;
  if (strings.length <= 1) {
    return strings.join('');
  } 
  if (!docsPlatform) {
    return null;
  }
  return strings.join(docsPlatform);
}
```

As an aside you can actually use tagged templates for this, if you feel adventurous. 

```js
const docsLink = docsUrl`https://docs.sentry.io/platforms/${project.platform}/performance/`;
```

Which would mean anywhere you had a project or project platform you could just generate the string in place. 

Anyway, this is just for fun though, this doesn't block the PR at all 😆

Reply:
Other than feeling adventurous, is there any other reason to prefer tagged templates? I personally find them harder to understand.

Reply:
I think my main issue with using tagged templates in this situation is that in the event that the platform does not support performance, we return null, and the tagged template notation (at least to me) feels like it should not be returning `null`. But if the return value is always the templated filled in, I agree that it might be nicer.

For the time being, I think I'll leave it as is since I don't think it'll be a major change if it comes to it.